app/Http/Controllers/AssetsController.php

```diff
@@ -1307,10 +1307,14 @@ public function auditStore(Request $request, $id)
 
     public function getRequestedIndex($id = null)
```

Comment:
I think this `$id` should be `$user_id` to avoid confusion - we're in an AssetsController but we're asking for assets for a particular `user_id`.